workers/repomirrorworker/__init__.py

```diff
@@ -31,6 +31,9 @@
     "number of repositories in the database that have not yet been mirrored",
 )
 
+# Used only for testing - should not be set in production
+TAG_ROLLBACK_PAGE_COUNT = app.config.get("REPO_MIRROR_TAG_ROLLBACK_PAGE_COUNT", 100)
```

Comment:
I don't see where it's being used in your tests. Do we need it?
You could also use `IS_TESTING`, I think the configuration is, for test specific values.

Reply:
Yeah i have this set for QE to use. Since the problem only occurs when tags reach the limit I was going to ask them to set the limit to be low and confirm the rollback still works.

Reply:
Ok. Small nit: can you rename it page `SIZE` instead of page count?

Reply:
Done